src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/movements/ExternalMovementsMigrationService.kt

```diff
@@ -149,7 +148,7 @@ class ExternalMovementsMigrationService(
     )
   }
 
-  private fun OffenderTemporaryAbsencesResponse.buildMappings(prisonerNumber: String, migrationId: String) = TemporaryAbsencesPrisonerMappingDto(
+  private fun OffenderTemporaryAbsencesResponse.buildMappings(prisonerNumber: String, migrationId: String, dpsResponse: MigrateTapResponse) = TemporaryAbsencesPrisonerMappingDto(
```

Comment:
The DPS response contains mappings of all DPS ids to NOMIS ids. We extract these from the DPS response to add to our mappings.